sassdocs_helpers.rb

```diff
@@ -0,0 +1,110 @@
+require 'json'
+
+module SassdocsHelpers
+    def format_sassdoc_data(data)
+        raise "No data in data/sassdocs.json, run `npm install` to generate." unless data.respond_to?(:sassdoc)
+        data.sassdoc
+            .select { |doc|
+                # Remove private API entries
+                doc.access == 'public'
+            }
+            .select { |doc|
+                # Remove vendored files, for example SassMQ
+                !doc.file.path.start_with?('vendor')
+            }
+            .group_by {
+                # Group the docs by their 'group', for example 'Settings' or 'Helpers'
+                |doc| doc.group.first
+            }
+    end
+    def mixin_trailing_code(code)
+        # Some mixins can have content passed inside.
+        # For example the `govuk-if-ie8` function:
+        # @include govuk-if-ie8 {
+        #  //..
+        # }
+        acceptsContent = code.include?('@content')
+        if acceptsContent
+            " {\n  //..\n}"
+        else
+            ";"
+        end
+    end
+    def inline_parameters(parameters)
+        # Generate the possible paramters that could be passed into a function or mixin.
+        # For example with the `govuk-colour` function: `$colour, $legacy`
+        # If the function / mixin has default values output them too, for example `$important: true`
+        parameters.map.with_index { |param, index|
+            output = "$#{param.name}"
+            if param['default']
+                output << ": "
+                if param.type == 'String'
+                    output << '"' + param['default'] + '"'
+                else
+                    output << param['default']
+                end
+            end
+            output
+        }.join(', ')
+    end
+    def parameters_table(parameters)
+        # Clone the parameters so we don't overwrite the original data when we map over it.
+        parameters.deep_dup.map { |param|
+            # If cell contains no data hide dash from assistive technology
+            no_data = "<span aria-hidden='true'>—</span>"
```

Comment:
Again minor, but can we flip the quotes so we get double quotes on the HTML attributes consistently?

Reply:
Will enable linting in line with tech-docs-gem to sort this.